src/main/java/Duke.java

```diff
@@ -1,10 +1,93 @@
+import java.util.Scanner;
+
+
 public class Duke {
+    public static void hi() {
+        System.out.println("Hello I'm Duke");
+        System.out.println("What can I do for you?");
+    }
+
+    public static void bye() {
```

Comment:
The name of the method have to be a verb instead:
https://se-education.org/guides/conventions/java/basic.html#naming